services/minespace-web/common/constants/actionTypes.js

```diff
@@ -85,6 +85,7 @@ export const STORE_VARIANCE_DOCUMENT_CATEGORY_OPTIONS = "STORE_VARIANCE_DOCUMENT
 // Project Summaries
 export const STORE_PROJECT_SUMMARIES = "STORE_PROJECT_SUMMARIES";
 export const STORE_PROJECT_SUMMARY = "STORE_PROJECT_SUMMARY";
+export const CLEAR_PROJECT_SUMMARY = "STORE_PROJECT_SUMMARY";
```

Comment:
it would need to be changed to 
export const CLEAR_PROJECT_SUMMARY = "CLEAR_PROJECT_SUMMARY"